internal/outputs/http.go

```diff
@@ -44,14 +48,16 @@ func postRequest(url string, key string, data []byte) error {
 	resp, err := client.Do(req)
 	if err != nil {
 		load.Logrus.WithError(err).Error("http: failed to send")
+		return errors.Wrap(err, "http: failed to send")
 	}
-	if resp == nil {
-		return fmt.Errorf("http: response nil")
-	}
-	defer resp.Body.Close()
+
+	defer func(body io.ReadCloser) {
+		_, _ = ioutil.ReadAll(body)
+		_ = body.Close()
+	}(resp.Body)
 
 	if resp.StatusCode > 299 || resp.StatusCode < 200 {
-		err = fmt.Errorf("http: post failed, status code: %d", resp.StatusCode)
+		return fmt.Errorf("http: post failed, status code: %d", resp.StatusCode)
```

Comment:
why this change?

Reply:
Because it basically did that two lines down after the `if` statement. Felt it was more obvious this was the return value here as we never process `err` after this.